pyabc/sampler/base.py

```diff
@@ -14,11 +15,15 @@ class Sample:
     record_rejected: bool
         Whether to record rejected particles as well, along with accepted
         ones.
+    ok: bool
```

Comment:
what about rename to, e.g., `succeeded` or `status_flag` or something more informative than `ok`

Reply:
I used ok (a.k.a. "all correct") as a short intuitive expression (``if sample.ok`` reads better than ``if sample.succeeded``, because it would be the sampler that succeeded, not the sample ..., I would say ...), and as a general flag that can be set to False when something goes wrong. But I can rename if your prefer ...